packages/@okta/vuepress-theme-prose/enhanceApp.js

```diff
@@ -1,14 +1,21 @@
-import VueSelect from 'vue-select';
-import pageComponents from '@internal/page-components'
-import 'bootstrap/dist/css/bootstrap-grid.css';
-import PortalVue from 'portal-vue';
+import VueSelect from "vue-select";
+import "bootstrap/dist/css/bootstrap-grid.css";
+import PortalVue from "portal-vue";
 
-export default ({
-  Vue,
-  options,
-  router,
-  siteData
-}) => {
+function scrollToAnchor(to) {
+  const targetAnchor = to.hash.slice(1);
+  const targetElement = document.getElementById(targetAnchor);
+
+  if (targetElement) {
+    const scrollToPosition = targetElement.offsetTop - 32;
```

Comment:
What is 32 and will it work when banner is hidden?
I guess that's `LAYOUT_CONSTANTS.ANCHOR_TOP_MARGIN`

Reply:
Yes, It will work when the banner is hidden. The 32 is `LAYOUT_CONSTANTS.ANCHOR_TOP_MARGIN`.

Reply:
Can you please update it to `LAYOUT_CONSTANTS.ANCHOR_TOP_MARGIN`?

Reply:
I'll not be able to access the constants from .vue file here so, I left it as 32. I can create a separate const file for this.